### ISSUE TITLE
Failed to compile using "<<< >>>" syntax

### ISSUE THREAD

**Issue by x3ccd4828** [ROCm-Developer-Tools/HIP#788]
According to the HIP documentation the "<<< >>>" syntax is supported, however I get a compile error "error:expected expression". 
Is there a compile flag I need?
hipcc -g   -c -o <file_name>.o <file_name>.cpp

**Comment by mangupta** [comment]
Could you point me to where it says that in the documentation? I will remove or reword it to make it clear that it is not supported. I could find two references in the documentation:
1. https://github.com/ROCm-Developer-Tools/HIP/blob/master/docs/markdown/hip_porting_driver_api.md#cumodule-api where it clearly mentions the syntax is only supported on nvcc.
2. https://github.com/ROCm-Developer-Tools/HIP/blob/master/docs/markdown/hip_porting_driver_api.md#kernel-launching where it mentions that the syntax is supported when using hip-clang as the compiler.
FYI the hipify tools also convert the ```<<< >>>``` syntax to ```hipLaunchKernelGGL```.

**Comment by x3ccd4828** [comment]
Thanks for clarifying I missed the fact that it's only supported on nvcc.
